assets/js/util/index.js

```diff
@@ -857,6 +857,15 @@ export function getDateRangeFrom() {
 	return getCurrentDateRange().replace( 'Last ', '' );
 }
 
+/**
+ * Gets the current dateRange slug.
+ *
+ * @return {string} the date range slug.
+ */
+export function getCurrentDateRangeSlug() {
+	return stringToSlug( getCurrentDateRange() );
```

Comment:
This still makes me cringe (slug based on translatable string), but okay since #447 will fix it.

Reply:
Yeah, I know 😬